doc/Language/about.pod6

```diff
@@ -153,12 +153,12 @@ limitations, special steps have to be taken to use B<V< X<> >> with other
 formatting codes; for example:
 
 =begin code :lang<pod6>
-=item X<B<foo>|foo> a fancy subroutine
+=item X<B<foo>|Subroutines,foo> a fancy subroutine
```

Comment:
Yes, but rules are rules, why not show them in examples?

Reply:
Because examples should only illustrate what they do illustrate, not something else. This is simply an example of formatting codes within indexing marks. If you want to illustrate categories, that's *very* fine. not in this case. And if you want to illustrate there are no first-level terms, explain as much below.

Reply:
You are right. We need a ticket then, to teach Documentable to skip `X<>` elements if they are placed within `code`, then this can be removed.

Reply:
Sorry, what's Documentable got to do with this? Did this generate an indexed term?

Reply:
Documentable is the current tool used to create search indexes out of Pod. There might be other tools too, I used Documentable as it 1)exists; 2)works well after some improvements.

So right now we have a test that walks through all the anchors and if it sees some that is bogus, it causes the test to fail, so to make it work we need to change the test and how this case (anchor in a code) is processed in Documentable.